site/src/components/OrganizationAutocomplete/OrganizationAutocomplete.tsx

```diff
@@ -51,11 +70,20 @@ export const OrganizationAutocomplete: FC<OrganizationAutocompleteProps> = ({
     750,
   );
 
+  // If an authorization check was provided, filter the organizations based on
+  // the results of that check.
+  let options = organizationsQuery.data ?? [];
+  if (check) {
```

Comment:
What is an organization check?

Reply:
if you don't pass in a check, you just get all of the organizations